backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/modelvalidator/ConfigValidator.java

```diff
@@ -113,8 +115,8 @@ protected void registerInterceptors(final IModelValidationEngine engine)
 		// engine.addModelValidator(new C_Invoice()); is now a spring component
 		engine.addModelValidator(new AD_Note());
 		engine.addModelValidator(new C_OrderLine());
-		engine.addModelValidator(new C_Order());
-		engine.addModelValidator(new M_InOut());
+		engine.addModelValidator(new C_Order(SpringContextHolder.instance.getBean(ApprovedForInvoicingService.class)));
```

Comment:
Please modernize these two to be a component, so that spring can inject the ApprovedForInvoicingService itself.
boyscout-rule